crates/context/primitives/src/client/external/proxy.rs

```diff
@@ -163,4 +163,20 @@ impl ExternalProxyClient<'_, '_> {
 
         Ok(approvals.num_approvals)
     }
+
+    pub async fn get_context_storage_entries(
```

Comment:
we can only access this through the `ContextClient`, so we don't need context in the name anymore 

Let's try `get_external_storage_entries`